internal/powerplatform/provider.go

```diff
@@ -181,6 +181,14 @@ func (p *PowerPlatformProvider) Configure(ctx context.Context, req provider.Conf
 		clientSecret = config.ClientSecret.ValueString()
 	}
 
+	useOidc := false
+	_, envUseOidcExists := os.LookupEnv("POWER_PLATFORM_USE_OIDC")
+	if config.UseOidc.IsNull() {
+		useOidc = envUseOidcExists
+	} else {
+		useOidc = config.UseOidc.ValueBool()
+	}
```

Comment:
Why is "exists" in the name of this variable?  It's not a actually if it exists.  It's the value of the environment variable.
```suggestion
	_, envUseOidc := os.LookupEnv("POWER_PLATFORM_USE_OIDC")
	if config.UseOidc.IsNull() {
		useOidc = envUseOidc
	} else {
		useOidc = config.UseOidc.ValueBool()
	}
```

Reply:
fair 👍 made the changes..